packages/generator-electrode/generators/app/templates/_package.json

```diff
@@ -35,7 +35,8 @@
     "electrode-server": "^1.0.0",
     "electrode-static-paths": "^1.0.0",<% } else { %>
     "express": "^4.0.0",<% } %> <% if (isPWA) { %>
-    "react-notify-toast": "^0.1.3",<% } %>
+    "react-notify-toast": "^0.1.3",<% } if (isAutoSSR) {%>
+    "electrode-auto-ssr": "git+https://github.com/animesh10/electrode-auto-ssr.git", <% } %>
```

Comment:
this needs to be added to https://github.com/electrode-io/electrode/blob/master/gulpfile.js#L17 for build CI test.

Also for now, change this to the version you expected to publish electrode-auto-ssr as.  ie. `^1.0.0`.  The CI test will automatically change it to point to the local copy.  Once PR is merged, we can then publish electrode-auto-ssr.